src/kibana/components/vislib/lib/y_axis.js

```diff
@@ -69,24 +99,9 @@ define(function (require) {
      * @param yMax
      * @returns {*[]}
      */
-    YAxis.prototype.returnLogDomain = function (yMin, yMax) {
-      if (yMin < 0 || yMax < 0) return this.throwCannotLogScaleNegVals();
-      return [Math.max(1, yMin), yMax];
-    };
-
-    /**
-     * Returns the domain, i.e. the extent of the y axis
-     *
-     * @param scale {String} Kibana scale
-     * @param yMin {Number} Y-axis minimum value
-     * @param yMax {Number} Y-axis maximum value
-     * @returns {*[]}
-     */
-    YAxis.prototype.getDomain = function (scale, yMin, yMax) {
-      if (scale === 'log') return this.returnLogDomain(yMin, yMax); // Negative values cannot be displayed with a log scale.
-      if (yMin === 0 && yMax === 0) return this.throwNoResultsError(); // yMin and yMax can never both be equal to zero
-
-      return [Math.min(0, yMin), Math.max(0, yMax)];
+    YAxis.prototype._logDomain = function (min, max) {
+      if (min < 0 || max < 0) return this._throwCannotLogScaleNegVals();
```

Comment:
Do you mean to display the min and max in the error message or display the min and max in the input box in the options menu? If its the latter, currently its not doable.